### ISSUE TITLE
Change Door Lock Code

### ISSUE THREAD

**Issue by johnkristijan** [persandstrom/python-verisure#114]
Thanks for creating this python repo ;)

I was testing it out and did not find a way to change the 6 digit PIN code that I have set on my Yale Doorman Lock (connected via Verisure).

Is there any way to change the doorlock code?

**Comment by persandstrom** [comment]
Hi,

No, you can't change codes. I think It would be possible to add for someone who has a yale lock thought. If it can be done from the app it's very likely that python-verisure can add it. May I ask what your use case is, I'm curious? 🙂

**Comment by johnkristijan** [comment]
Ok. Thanks for clarifying. I have created a booking web page for a local community organization where we hire out 2 apartments with Yale doorman locks installed. So now the booking process is all automated with payment etc. but we are lacking an automated way of changing the customers’ PIN codes that they use during their stay. Now it’s changed manually in the Verisure app on someone’s smartphone every time someone books... we have asked Verisure several times if they have an open api or if they are planning to, but they’re quite reluctant and claims top level security is the reason for not opening the api for the public. Any tips in changing the codes will be greatly appreciated!

**Comment by bengtdg** [comment]
Seeing this is an old thread, but i have the same challenge as johnkristija. 
Is this function available now?

**Comment by persandstrom** [comment]
No, it's not available, and to implement that would most likely require quite a lot of work since changing codes is done through the mypages and not via the app.

**Comment by bengtdg** [comment]
> No, it's not available, and to implement that would most likely require quite a lot of work since changing codes is done through the mypages and not via the app.

Suspected that. Thanx for your answer ;)
